packages/vite-plugin-svelte/src/utils/preprocess.ts

```diff
@@ -94,7 +95,9 @@ function createInjectScopeEverythingRulePreprocessorGroup(): PreprocessorGroup {
 			s.append(' *{}');
 			return {
 				code: s.toString(),
-				map: s.generateDecodedMap({ source: filename, hires: true })
+				map: filename
```

Comment:
i think filename is always set here, but just to be sure it would be better to always generate the map but leave source undefined ` s.generateDecodedMap({ source: filename ? path.basename(filename) : undefined , hires: true })`